### PR TITLE
Create PHYS211 

### PR DESCRIPTION
**Pull request by taoweih**

### CODE REVIEW

File: docs/Course Wiki/PHYS Course Offerings/PHYS211.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+ECE majors typically take PHYS211 within the first two semesters, most commonly during the first semester.
+
+## Course Structure
+
+PHYS211 holds two lectures each week. There are prelectures and checkpoints before each lecture. Attendance for the lectures is required using iClickers. PHYS211 has a required discussion section, which meets weekly. During the discussion, students work on a problem set related to the topics covered in the lectures. At the end of the discussion, there is a weekly quiz except during the midterm weeks. There is also a required weekly lab section. During the lab, students work on a lab related to the topics covered in the lectures. There are prelabs due before the labs and a lab report due at the end of the lab session.
+
+## Instructors
+
+Recently, this course has been taught by Professor Tim Stelzer and Professor Jorge Leite Noronha Jr.
```
Comment (lnguy7) [inline docs/Course Wiki/PHYS Course Offerings/PHYS211.md]: Do we have any info on these two profs?

File: docs/Proficiency Exams/CS173.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# CS173
```
Comment (lnguy7) [inline docs/Proficiency Exams/CS173.md]: Make sure to label as 128 not173

**Review by lnguy7 — CHANGES_REQUESTED** [review]

**Review by lnguy7 — APPROVED** [review]

File: docs/Course Wiki/PHYS Course Offerings/PHYS211.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+Between prelectures, lectures, labs and discussions, each topic will be covered multiple times. If students are confused about a certain topic, it is best to go through all the different components of the class. The formula sheet for this class is quite good. Understanding when to use each formula is very helpful for exams. The practice exams are very similar to the actual exams, they are a great tool to study. 
+
+## Life After
+
+Students will generally take [PHYS212](PHYS212.md) the semester after taking PHYS211. PHYS212 is not directly related to 211 but is required for both EEs and CEs. 
+
+## Infamous Topics
+
+Ratations and Waves can be difficult for students who are seeing them for the first time. Understanding of how to use the formulas on the formula sheet is extremely helpful for dealing with those topics in exams.
```
Comment (aidanrodgers) [inline docs/Course Wiki/PHYS Course Offerings/PHYS211.md]: "Ratations and Waves" -> "Rotations and waves", "Understanding of" -> "Understanding"

File: docs/Course Wiki/PHYS Course Offerings/PHYS211.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+- Collisions, Impulse and Reference Frames
+- Ratations, Torque, Angular Momentum
+- Simple Harmonic Motion, Pendula, Waves
+- Fluid Statics, Fluid Dynamics
+
+## Prerequisites
+
+- [MATH231](MATH231.md) (corequisite)
+
+The official prerequisites to PHYS211 is credit or concurrent registration in [MATH231](MATH231.md)(While MATH231 is listed as a corequisite for PHYS211, no heavy calculation will be used in PHYS211).
```
Comment (aidanrodgers) [inline docs/Course Wiki/PHYS Course Offerings/PHYS211.md]: "The official prerequisites to PHYS211 is" -> "The official prerequisites to PHYS211 are"
Make "while" lowercase, put a space after MATH231.

**Review by aidanrodgers — CHANGES_REQUESTED** [review]
Minor changes.

File: docs/Course Wiki/PHYS Course Offerings/PHYS211.md [hunk 5]
```diff
@@ -44,7 +44,7 @@ Students will generally take [PHYS212](PHYS212.md) the semester after taking PHY
 
 ## Infamous Topics
 
-Ratations and Waves can be difficult for students who are seeing them for the first time. Understanding of how to use the formulas on the formula sheet is extremely helpful for dealing with those topics in exams. 
+Ratations and waves can be difficult for students who are seeing them for the first time. Understanding how to use the formulas on the formula sheet is extremely helpful for dealing with those topics in exams.
```
Comment (aidanrodgers) [inline docs/Course Wiki/PHYS Course Offerings/PHYS211.md]: "Rotations" spelled incorrectly

**Review by aidanrodgers — CHANGES_REQUESTED** [review]
Last thing, sorry

**Review by aidanrodgers — APPROVED** [review]
Looks good, thanks!

**Pull request merged**